### ISSUE TITLE
Does this tool really work?

### ISSUE THREAD

**Issue by R3LEASY** [Leeon123/CC-attack#119]
Like does it work? Simple, yes or no

**Comment by Pochinki98** [comment]
Yea but sometimes it depends on your server lah...

**Comment by peterpt** [comment]
yes , it works perfectly , i tried it on one of my remote servers , while i was attacking it i had also a remote shell to that server with htop watching cpu usage on server from attack , the script worked perfectly , but if you run in on several instances , like 1 for http attack , other for socks4 and last instance for socks5 then it will work perfectly . It does not mean you may be able to take that server down , but it will exhaust the webserver resources a lot . Also depends on your proxy list .

**Comment by Vip3rLi0n** [comment]
Yes it works. Also, I have updated some codes and made a lot of changes as well especially proxylist. Feel free to check my repo.
https://github.com/Vip3rLi0n/CC-attack
